code/modules/vending/_vending.dm

```diff
@@ -104,14 +106,14 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	// so if slogantime is 10 minutes, it will say it at somewhere between 10 and 20 minutes after the machine is crated.
 	last_slogan = world.time + rand(0, slogan_delay)
 	power_change()
-
-	if(mapload) //check if it was initially created off station during mapload.
-		if(!is_station_level(z))
-			onstation = FALSE
-			if(circuit)
-				circuit.onstation = onstation //sync up the circuit so the pricing schema is carried over if it's reconstructed.
-	else if(circuit && (circuit.onstation != onstation)) //check if they're not the same to minimize the amount of edited values.
-		onstation = circuit.onstation //if it was constructed outside mapload, sync the vendor up with the circuit's var so you can't bypass price requirements by moving / reconstructing it off station.
+	if(!onstation_override) //overrides the checks if the variable is set to false. 
```

Comment:
```DM
if(onstation_override)
    onstation = TRUE
    return
//old, not nested code here
```
???

Reply:
It isn't supposed to be there, this is a weird git glitch

Reply:
![image](https://user-images.githubusercontent.com/26174449/61910073-a20b7a00-af01-11e9-873d-1ceb2b6e54a7.png)

Reply:
This avoids running all that code for no reason.